compiler/build.gradle

```diff
@@ -75,6 +75,7 @@ model {
   binaries {
     all {
       if (toolChain in Gcc || toolChain in Clang) {
+        cppCompiler.define("GRPC_VERSION", version)
```

Comment:
Ah, makes sense. Thanks! I figure it won't be a big deal if we leave it like this then?